### PR TITLE
Introduced $request_start_time and updated $random_N

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

**Review by burtonquantil — DISMISSED** [review]
You can merge after considering the suggestion I made.

File: en/docs/edge-logic/built-in-variables.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ If you use the CDN Pro portal to edit the property, typing `$` at a position whe
...
 | <span id="is_args"></span><span class="var">$is_args</span> | empty or "?", when query string exists | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
-| <span id="msec"></span><span class="var">$msec</span> |<span class="badge small" title="numerical value">#</span> current time in seconds with milliseconds resolution | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
+| <span id="msec"></span><span class="var">$msec</span> |<span class="badge small" title="numerical value">#</span> current unix time in seconds with milliseconds resolution | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="sec"></span><span class="var">$sec</span> |<span class="badge small" title="numerical value">#</span> current unix time in integer seconds | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="origin_host"></span><span class="var">$origin_host</span> | origin's hostname | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (marcfishelquantil) [inline en/docs/edge-logic/built-in-variables.md:37]: Please consider capitalizing Unix.

File: en/docs/edge-logic/built-in-variables.md [hunk 2]
```diff
@@ -43,13 +43,14 @@ If you use the CDN Pro portal to edit the property, typing `$` at a position whe
...
 | <span id="request_cpu_time"></span><span class="var">$request_cpu_time</span> |<span class="badge small" title="numerical value">#</span> CPU time, in nanoseconds, spent on processing the request by the ES stage (when used in the Edge Logic), or by the LB stage (when used in the real-time log) | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
-| <span id="request_end_time"></span><span class="var">$request_end_time</span> |<span class="badge small" title="numerical value">#</span> the time needed for CDN Pro to receive 'fully' the request from the Client and be ready to process/forward it, in seconds. Another variable request_end_time_ms shares the same meaning but in milliseconds.  | <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
+| <span id="request_start_time"></span><span class="var">$request_start_time</span> |<span class="badge small" title="numerical value">#</span> the unix time (in seconds with milliseconds resolution) when the first byte of the request is received from the client. | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (marcfishelquantil) [inline en/docs/edge-logic/built-in-variables.md:52]: Please consider capitalizing Unix.

File: zh/docs/edge-logic/built-in-variables.md [hunk 3]
```diff
@@ -31,7 +31,7 @@
 | <span id="ignored_body_in_cache_key"></span><span class="var">$ignored_body_in_cache_key</span> | 空字符串，当 [`proxy_request_body_in_cache_key`](/docs/edge-logic/supported-directives#proxy_request_body_in_cache_key) 被设为 `off`, 或者 '0' 当请求正文的哈希值被加到了缓存key, 否则值为 '1'。| <span class="badge green">Yes</span> | <span class="badge yellow">No</span> |
 | <span id="invalid_referer"></span><span class="var">$invalid_referer</span> | 用于标记请求 Referer 的合法性，如“ Referer ”被判定为合法则值为空 ; 否则值为 “1” | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="is_args"></span><span class="var">$is_args</span> | 当客户端请求携带问号后参数时，值为 "?" ；否则值为空 | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
-| <span id="msec"></span><span class="var">$msec</span> |<span class="badge small" title="numerical value">#</span> 当前 unix 时间戳，精度到毫秒 | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (marcfishelquantil) [inline zh/docs/edge-logic/built-in-variables.md:34]: Please consider capitalizing Unix.

File: zh/docs/edge-logic/built-in-variables.md [hunk 4]
```diff
@@ -31,7 +31,7 @@
...
+| <span id="msec"></span><span class="var">$msec</span> |<span class="badge small" title="numerical value">#</span> 当前 unix 时间戳，以秒为单位，精度到毫秒 | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (marcfishelquantil) [inline zh/docs/edge-logic/built-in-variables.md:34]: Please consider capitalizing Unix.

File: zh/docs/edge-logic/built-in-variables.md [hunk 5]
```diff
@@ -31,7 +31,7 @@
...
 | <span id="sec"></span><span class="var">$sec</span> |<span class="badge small" title="numerical value">#</span> 当前 unix 时间戳，精度到秒 | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (marcfishelquantil) [inline zh/docs/edge-logic/built-in-variables.md:35]: Please consider capitalizing Unix.

File: zh/docs/edge-logic/built-in-variables.md [hunk 6]
```diff
@@ -42,13 +42,14 @@
...
 | <span id="request"></span><span class="var">$request</span> | 完整的 http 请求行 | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="request_cpu_time"></span><span class="var">$request_cpu_time</span> |<span class="badge small" title="numerical value">#</span> 处理该请求时所消耗的CPU时间, 在Edge Logic 中被使用时仅包含 ES 层的执行消耗，当在 LB Logic 或实时日志中被使用时仅包含负载均衡层的总执行消耗。单位为纳秒。| <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
-| <span id="request_end_time"></span><span class="var">$request_end_time</span> |<span class="badge small" title="numerical value">#</span> CDN Pro "全量" 接收来自客户端的请求并准备好处理/转发它所需的时间，单位为秒。同时支持毫秒单位的变量request_end_time_ms。  | <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
+| <span id="request_start_time"></span><span class="var">$request_start_time</span> |<span class="badge small" title="numerical value">#</span> CDN Pro接收到客户请求第一个字节的 unix 时间戳，以秒为单位，精度到毫秒 | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (marcfishelquantil) [inline zh/docs/edge-logic/built-in-variables.md:51]: Please consider capitalizing Unix.

**Review by marcfishelquantil — COMMENTED** [review]
Please see my comments.

**Comment by nibinqtl** [comment]
@burtonquantil @marcfishelquantil , made some more changes. Please review again.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Review by marcfishelquantil — COMMENTED** [review]
Approved.

**Pull request merged**